src/constants.js

```diff
@@ -1,16 +1,23 @@
 // @flow
 
 declare var __DEV__: ?string
+declare var SC_DISABLE_SPEEDY: ?boolean
 
 export const SC_ATTR =
   (typeof process !== 'undefined' && process.env.SC_ATTR) ||
   'data-styled-components'
+
 export const SC_STREAM_ATTR = 'data-styled-streamed'
+
 export const CONTEXT_KEY = '__styled-components-stylesheet__'
 
 export const IS_BROWSER =
   typeof window !== 'undefined' && 'HTMLElement' in window
 
 export const DISABLE_SPEEDY =
   (typeof __DEV__ === 'boolean' && __DEV__) ||
+  (typeof SC_DISABLE_SPEEDY === 'boolean' && SC_DISABLE_SPEEDY) ||
   process.env.NODE_ENV !== 'production'
+
+// Shared empty execution context when generating static styles
+export const STATIC_EXECUTION_CONTEXT = {}
```

Comment:
This is unused, right?

Reply:
Yep, that shouldn't have come over. @platypusrex is out today, so I may just submit a new PR that fixes this so we can keep the ball rolling if that sounds alright with you?

Reply:
Moved to #2272, so we can close this PR now 👍